org.hl7.fhir.dstu2016may/src/main/java/org/hl7/fhir/dstu2016may/utils/client/FHIRToolingClient.java

```diff
@@ -87,6 +88,9 @@
 	public static final String DATE_FORMAT = "yyyy-MM-dd";
   public static final String hostKey = "http.proxyHost";
 	public static final String portKey = "http.proxyPort";
+  private static final int TIMEOUT_NORMAL = 1;
+  private static final int TIMEOUT_OPERATION = 2;
+  private static final int TIMEOUT_OPERATION_LONG = 3;
```

Comment:
Yeah, this is used in a few different places. Putting it in it's own class as an enum is the right approach.